lib/macho/macho_file.rb

```diff
@@ -419,7 +419,7 @@ def delete_rpath(path, _options = {})
       raise RpathUnknownError, path if rpath_cmds.empty?
 
       # delete the commands in reverse order, offset descending.
-      rpath_cmds.reverse_each { |cmd| delete_command(cmd) }
+      delete_command(rpath_cmds.last)
```

Comment:
Hmm, now that we're only deleting one at a time, maybe we should delete the first instead of the last (since that's what I suspect `install_name_tool` does).

That in turn makes everything a little more efficient, since we can switch from `#select` to `#find` for finding the first RPATH. WDYT?

Reply:
Sure, makes sense to me.